### PR TITLE
Setting up a project with react native

### LINKED ISSUE #1: Setup the project
This task includes choosing a technology and setting up the project.

### PR DESCRIPTION
**Pull request by dagmaraskulimowska**
Close #1

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
```
Comment (KacperWojdak) [inline App.tsx:7]: No need for this comment

File: App.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    marginTop: 8,
+    fontSize: 18,
+    fontWeight: '400',
+  },
+  highlight: {
+    fontWeight: '700',
+  },
+});
+
+export default App;
```
Comment (KacperWojdak) [inline App.tsx:118]: Missing one empty line under

File: __tests__/App.test.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/**
+ * @format
+ */
+
```
Comment (KacperWojdak) [inline __tests__/App.test.tsx:4]: Comment

File: android/app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    // extraPackagerArgs = []
+
+    /* Hermes Commands */
+    //   The hermes compiler command to run. By default it is 'hermesc'
+    // hermesCommand = "$rootDir/my-custom-hermesc/bin/hermesc"
+    //
+    //   The list of flags to pass to the Hermes compiler. By default is "-O", "-output-source-map"
+    // hermesFlags = ["-O", "-output-source-map"]
+
+    /* Autolinking */
```
Comment (KacperWojdak) [inline android/app/build.gradle:53]: Ulala, I guess it's not important

File: android/app/src/main/java/com/kebabmobile/MainActivity.kt
Comment (KacperWojdak) [inline android/app/src/main/java/com/kebabmobile/MainActivity.kt:1]: Comments

File: android/app/src/main/res/drawable/rn_edit_text_material.xml
Comment (KacperWojdak) [inline android/app/src/main/res/drawable/rn_edit_text_material.xml:1]: Comments

File: android/gradlew
Comment (KacperWojdak) [inline android/gradlew:1]: Comments

File: ios/KebabMobile/AppDelegate.mm [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+#import <React/RCTBundleURLProvider.h>
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  self.moduleName = @"KebabMobile";
+  // You can add your custom initial props in the dictionary below.
+  // They will be passed down to the ViewController used by React Native.
```
Comment (KacperWojdak) [inline ios/KebabMobile/AppDelegate.mm:11]: Here

**Review by KacperWojdak — CHANGES_REQUESTED** [review]
It's basically clean. 😊

**Review by KacperWojdak — APPROVED** [review]

**Review by KacperWojdak — APPROVED** [review]

**Review by KacperWojdak — APPROVED** [review]
Yeah I like it

File: .eslintrc.js
Comment (PiotrFedak) [inline .eslintrc.js:1]: This file should work but can be done better with:

```
 extends: [
    '@react-native-community', 
  ],
```
  
`plugins: ['react', 'react-native'],`

Or even the rules

**Review by PiotrFedak — COMMENTED** [review]
It would be nice to set up a CI/CD workflow to linting code since you have Eslint

File: .eslintrc.js
Comment (TomaszBarczycki) [reply .eslintrc.js:1]: It's done, but not with the community version - it caused errors

**Review by PiotrFedak — APPROVED** [review]

**Pull request merged**